src/main/java/com/genfood/foodgenback/endpoint/rest/model/LoggedUser.java

```diff
@@ -0,0 +1,15 @@
+package com.genfood.foodgenback.endpoint.rest.model;
+
+import java.io.Serializable;
+import lombok.Builder;
+import lombok.Data;
+
+@Builder
+@Data
+public class LoggedUser implements Serializable {
```

Comment:
Update this class to look like the specification.
![image](https://github.com/HEI-CleanCodeProject/FoodGen-Back/assets/98638893/ecbcaab2-5ca2-4515-8b30-20fd4b78ed4e)